Adyen/UI/Views/SubmitButton.swift

```diff
@@ -41,10 +64,8 @@ public final class SubmitButton: UIControl {
     
     internal lazy var backgroundView: BackgroundView = {
         let backgroundView = BackgroundView(
-            cornerRounding: style.cornerRounding,
-            borderColor: style.borderColor,
-            borderWidth: style.borderWidth,
-            color: style.backgroundColor
+            cornerRounding: buttonStyle.cornerRadius ?? .fixed(14.0),
```

Comment:
This is a magic number, it should come from global defaults and not be defined in specific classes